hedera-node/hedera-addressbook-service-impl/build.gradle.kts

```diff
@@ -21,6 +21,8 @@ plugins {
 
 description = "Default Hedera AddressBook Service Implementation"
 
+dependencies { implementation(project(":swirlds-platform-core")) }
+
```

Comment:
Is this necessary? I thought we were able to do this just by depending on the right classes in `module-info.java`

Reply:
Verified, we should be able to get rid of this. We can do it in a separate PR though since the checks are passing